src/main.cpp

```diff
@@ -269,6 +272,12 @@ void setup()
     powerStatus->observe(&power->newStatus);
     power->setup(); // Must be after status handler is installed, so that handler gets notified of the initial configuration
 
+    /*
+    * Move the scanning I2C device to the back of power initialization. 
+    * Some boards need to be powered on to correctly scan to the device address, such as t-beam-s3-core
+    */
+    scanI2Cdevice();
+    
```

Comment:
will this work with the AXP chips? Do they rely on this scan? Everything else comes in afterwards.

Reply:
Although the sensor uses Wire, while the PMU uses Wire1, their bus is not shared. If the PMU is not initialized before the sensor voltage is turned on, the scanI2Cdevice will be blocked because the wire cannot scan the device correctly, similar to T-beam's inability to initialize the PMU after turning off DC3

Reply:
I was considering the existing T-Beam with AXP192 too - but from the code the i2c detect for them is just cosmetic. So this is good to go!